api/HTMLMediaElement.json

```diff
@@ -960,7 +960,7 @@
               "version_added": "49"
             },
             "edge": {
-              "version_added": "12"
+              "version_added": "79"
```

Comment:
The "12" came from mirroring of IE data in https://github.com/mdn/browser-compat-data/pull/5566, which in turn came from https://github.com/mdn/browser-compat-data/pull/1166.

I've confirmed that http://mdn-bcd-collector.appspot.com/tests/api/HTMLMediaElement/disableRemotePlayback returns false in IE11. https://w3c.github.io/remote-playback/ is newer than IE11, so I think the data was just wrong.